### PR TITLE
move pom.xml from root directory to /zoo

### PR DESCRIPTION
**Pull request by Le-Zheng**

### CODE REVIEW

File: zoo/pom.xml [hunk 1]
```diff
@@ -4,14 +4,52 @@
...
-        <groupId>com.intel.analytics.zoo</groupId>
-        <version>0.10.0-SNAPSHOT</version>
-    </parent>
-
+    <groupId>com.intel.analytics.zoo</groupId>
     <artifactId>analytics-zoo-bigdl_${bigdl.version}-spark_${spark.version}</artifactId>
     <packaging>jar</packaging>
+    <version>0.10.0-SNAPSHOT</version>
+    
+    <description>A distributed analytics library for Apache Spark.</description>
```
Comment (jason-dai) [inline zoo/pom.xml:12]: Big Data AI platform for distributed TensorFlow and PyTorch on Apache Spark

**Comment by glorysdj** [comment]
please run the jenkins publish jobs and check all the artifacts.
and we need also to modify our release job since there is no zoo-parent now.

**Comment by Le-Zheng** [comment]
> please run the jenkins publish jobs and check all the artifacts.
> and we also need to modify our release job since there is no zoo-parent now.

Newly published artifacts(Fri Apr 09 03:42:33 UTC 2021): https://oss.sonatype.org/content/repositories/snapshots/com/intel/analytics/zoo/analytics-zoo-bigdl_0.12.1-spark_2.4.3/0.10.0-SNAPSHOT/

**Comment by Le-Zheng** [comment]
http://10.239.47.210:18888/view/ZOO-PR/job/ZOO-PR-Validation/5363/ + http://10.239.47.210:18888/job/ZOO-PR-Python-ExampleTests/2595/ + http://10.239.47.210:18888/job/ZOO-PR-Python-ExampleTests-py37/2244/ + http://10.239.47.210:18888/job/ZOO-PR-Scala-Spark-2.4-k216/3733/

**Review by glorysdj — APPROVED** [review]
LGTM

**Pull request merged**